components/omega/src/timeStepping/TimeStepper.cpp

```diff
@@ -114,13 +110,29 @@ int TimeStepper::init() {
    }
    TimeStepperChoice = getTimeStepperFromStr(TimeStepperStr);
 
-   TimeStepper::DefaultTimeStepper = create(
-       "Default", TimeStepperChoice, DefTend, DefAuxState, DefMesh, DefHalo);
+   TimeStepper::DefaultTimeStepper =
+       create("Default", TimeStepperChoice, nullptr, nullptr, nullptr, nullptr);
    DefaultTimeStepper->setTimeStep(TimeStep);
 
    return Err;
 }
 
+// Finish initialization of the default time stepper
+int TimeStepper::init2() {
+   int Err = 0;
+
+   DefaultTimeStepper->Mesh     = HorzMesh::getDefault();
+   DefaultTimeStepper->MeshHalo = Halo::getDefault();
+   DefaultTimeStepper->AuxState = AuxiliaryState::getDefault();
+   DefaultTimeStepper->Tend     = Tendencies::getDefault();
+
+   if (DefaultTimeStepper->Type == TimeStepperType::RungeKutta4) {
+      DefaultTimeStepper->finalizeInit();
+   }
```

Comment:
```suggestion
      DefaultTimeStepper->finalizeInit();
```
I think this is equivalent because by default this function does nothing ?

Reply:
Ha, I suppose so, will do this